app/services/decidim/erc/crm_authenticable/crm_authenticable_authorization_handler.rb

```diff
@@ -48,6 +54,33 @@ def ws_request_must_succeed
           errors.add(:base, I18n.t("connection_failed", scope: "crm_authenticable.errors")) if response[:error]
         end
 
+        def csv_request_must_succeed
+          return if errors.any?
+
+          file = Rails.application.secrets.csv_users_pre[:path]
```

Comment:
I don't see the `secrets` file modified taking this value from an env var

Reply:
this should be in the app, so it's correct

Reply:
There is in https://github.com/CodiTramuntana/decidim-erc-app/pull/79